Source/Core/VideoCommon/RenderBase.h

```diff
@@ -133,6 +134,8 @@ class Renderer
 	// Max height/width
 	virtual int GetMaxTextureSize() = 0;
 
+	static Common::Event s_screenshotCompleted;
```

Comment:
The reason I used an `Event` is because I need to cause a thread to block until the `Event` is set. Is there  a way to do this with just a `bool`?